src/main/java/frc/robot/constants/RobotConstants.java

```diff
@@ -19,12 +20,30 @@
         public double LEFT_ENCODER_TICKS_PER_METER;
         public double RIGHT_ENCODER_TICKS_PER_METER;
     }
+    
     public static class PIDConstants {
-        public PIDSettings exampleSettings;
+        public PIDSettings leftMotionProfilingSettings;
```

Comment:
Why do you declare PIDSettings which has kP, kV, kA and kS for each side and declare kV, kA and kS in MotionProfileingConstants?
Either you remove it from the constants or keep only keep kp